catalog/spatial/registry/registry-api-impl/src/main/java/org/codice/ddf/registry/api/impl/RegistryStoreImpl.java

```diff
@@ -204,4 +261,79 @@ public void setParser(Parser parser) {
         this.marshalConfigurator.addProperty(Marshaller.JAXB_FRAGMENT, true);
         this.parser = parser;
     }
+
+    public SourceMonitor registrySourceMonitor = new SourceMonitor() {
```

Comment:
Oh yeah, if this is moved inside init, then I can't access the private registryInfoQuery... I did change it to protected.